int32[] pose
---
bool set_ok